crates/matrix-sdk-indexeddb/src/crypto_store/migrations/v8_to_v10.rs

```diff
@@ -132,6 +135,11 @@ pub(crate) async fn migrate_data_before_v10_populate_inbound_group_sessions3(
         }
     }
 
+    // We have finished with the old store. Clear it, since it is faster to
+    // clear+delete than just delete. See https://www.artificialworlds.net/blog/2024/02/01/deleting-an-indexed-db-store-can-be-incredibly-slow-on-firefox/
```

Comment:
Haha, correct link is https://www.artificialworlds.net/blog/2024/02/02/deleting-an-indexed-db-store-can-be-incredibly-slow-on-firefox/ ^^ (cf https://github.com/matrix-org/matrix-rust-sdk/pull/3094).